system-tests/projects/session-and-origin-e2e-specs/cypress/e2e/session/session.cy.js

```diff
@@ -38,102 +38,6 @@ const sessionUser = (name = 'user0', cacheAcrossSpecs = false) => {
   })
 }
 
-describe('cross origin automations', function () {
```

Comment:
Are you planning to replace these tests that were deleted?

Reply:
The `clearStorage` and `getStorage` functions are well covered by the tests for the new commands. I moved the `setStorage` test into the driver.